tracker/tasks.py

```diff
@@ -23,6 +23,7 @@ def add_mailchimp_subscriber(email):
         not r.ok
         and r.json()['title'] != 'Member Exists'
         and 'looks fake or invalid' not in r.json()['detail']
+        and 'signed up to a lot of lists very recently' not in r.json()['detail']
     ):
-        logger.error(r.text)
+        logger.info(r.text)
```

Comment:
Why only log an `info`? I would think at least `warning` would be better, so we find out in Sentry about any other unexpected reasons that this failed.

Reply:
Logging warning and above creates a sentry issue. This particular problem was creating 2 sentry issues. 

Logging as info gives us access to the information as a bread crumb if the exception ends up being raised.

Reply:
Ah, I see.